tensorflow/core/kernels/mkl/mkl_conv_grad_filter_ops.cc

```diff
@@ -370,6 +370,14 @@ class MklConvCustomBackpropFilterOp
       // a correct way to get filter shape. These operator-specific calls
       // allow this class to handle this case.
       TensorShape src_tf_shape = MakeInputTfShape(context, src_tensor);
+      const string& op_type = this->type_string();
+      if ((op_type.find("3D") != std::string::npos) &&
```

Comment:
If the fix is only specific to `Conv3DBackpropFilterV2`, why don't we just check for the full string?
```suggestion
      if (op_type == "Conv3DBackpropFilterV2")
```

Reply:
@penpornk - thanks for the quick response. In the oneDNN enabled build, the op is replaced by mkl op, there are currently two mkl ops (_MklConv3DBackpropFilterV2, line 745, _MklNativeConv3DBackpropFilterV2, line 763) need to be checked. I used this checking instead of the exact op name similar as what is done in commit 174c5096 for the eigen kernel. 
Please let me know if you want me to make the change or stay with current code. Thanks

Reply:
Thank you for the explanation! I forgot about the native and non-native versions. Let's keep the current code. :)